SmartDeviceLink/SDLCarWindow.m

```diff
@@ -141,9 +144,12 @@ - (void)setRootViewController:(nullable UIViewController *)rootViewController {
         return;
     }
 
-    NSAssert((rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskPortrait ||
-              rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskLandscapeLeft ||
-              rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskLandscapeRight), @"SDLCarWindow rootViewController must support only a single interface orientation");
+    if (!self.allowMultipleOrientations
+        && (rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskPortrait ||
+            rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskLandscapeLeft ||
+            rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskLandscapeRight)) {
+        @throw [NSException sdl_carWindowOrientationException];
```

Comment:
An exception is being thrown when `allowMultipleOrientations` is `false` and the view controller only supports `UIInterfaceOrientationMaskPortrait`. The assert should only be thrown if  `allowMultipleOrientations` is `false` and the `supportedInterfaceOrientations` check returns `false` 
```
if (!self.allowMultipleOrientations
    && !(rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskPortrait ||
        rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskLandscapeLeft ||
        rootViewController.supportedInterfaceOrientations == UIInterfaceOrientationMaskLandscapeRight)) {
    @throw [NSException sdl_carWindowOrientationException];
}
```